realsense_camera/launch/f200_nodelet_default.launch

```diff
@@ -4,7 +4,34 @@
   <arg name="camera_type"  default="F200" /> <!-- Type of camera -->
   <arg name="serial_no"    default="" />
   <arg name="usb_port_id"  default="" /> <!-- USB "Bus#-Port#" -->
-  <arg name="manager"      value="$(arg camera)_nodelet_manager" />
+  <arg name="manager"      value="nodelet_manager" />
```

Comment:
I realize you didn't change this, but should this be default= instead of value=? If it is default, it can be overridden by the command line if it is value, it is essentially hard-coded to "nodelet_manager". See this: http://wiki.ros.org/roslaunch/XML/arg. Same would apply for all the camera types.